.coderabbit.yaml

```
@@ -0,0 +1,105 @@
+language: en
+
+reviews:
+  profile: "chill"
+  request_changes_workflow: true
+  auto_review:
+    enabled: true
+    drafts: false
+  auto_review:
```

YAMLlint (1.37.1)

[error] duplication of key "auto_review" in mapping

(key-duplicates)